packages/design-system/src/scalars/components/radio-group/radio/radio.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useId } from "react";
+import * as RadioGroupPrimitive from "@radix-ui/react-radio-group";
+import { Circle } from "lucide-react";
+import { FormLabel } from "@/scalars/components/form-label";
+import { cn } from "@/scalars/lib/utils";
+
+export interface RadioProps
+  extends React.ComponentPropsWithoutRef<typeof RadioGroupPrimitive.Item> {
+  className?: string;
+  description?: string;
+  hasError?: boolean;
+  id?: string;
+  label: string;
+  readOnly?: boolean;
```

Comment:
we should handle readonly from design as disabled, as it is the correct state in the HTML standard (readonly exist and it is valid but it is not the correct state for this one)

Reply:
Done